libs/common/src/constants/company-registration-number.ts

```diff
@@ -0,0 +1,15 @@
+export const COMPANY_REGISTRATION_NUMBER = {
+  REGEX: /^(([A-Z]{2}|[A-Z]\d{1}|\d{2})(\d{5,6}|\d{4,5}[A-Z]))$/,
```

Comment:
```suggestion
  REGEX: /^(([A-Z]{2}|[A-Z]\d{1}|\d{2})(\d{6}|\d{5}[A-Z]))$/,
```

Reply:
@abhi-markan just noticed, the current regex accepts 7 or 8 characters, but all Companies House registration numbers are 8 characters, so this is my proposed change.

Reply:
As discussed on Slack, I've made this change